src/components/header.tsx

```diff
@@ -64,6 +65,11 @@ function HeaderContent() {
             flexShrink={0}
             display={{ base: 'none', md: 'flex' }}
           />
+          <LanguageSwitcher
+            width='auto'
+            flexShrink={0}
+            display={{ base: 'none', md: 'flex' }}
+          />
```

Comment:
To be fair I would style the language switcher to look a little better, It looks a little too small. 
![image](https://user-images.githubusercontent.com/44305048/209997796-5be56d31-60cb-4ba1-a83a-ba35e6ca0fa0.png)

This is what I found from searching on the web.

Reply:
I will do so :) Glad you pointed it out. I was unsure whether to have text or icons.